manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -217,7 +217,7 @@ spec:
         path: nodeSpec.resources
         x-descriptors:
           - 'urn:alm:descriptor:com.tectonic.ui:resourceRequirements'
-      - description:
+      - description: The image to use for the deployment
         displayName: Elasticsearch Image
         path: nodeSpec.image
       - description:
```

Comment:
doesn't this also need a description then?

Reply:
yes.